### PR TITLE
Bring ansible installation on par with manual installation

### PR DESCRIPTION
**Pull request by rtorrero**
This PR applies the steps described in https://github.com/trento-project/docs/pull/35 to the automated Ansible playbooks. Changes:
 - The playbook aims now to install trento on SLES 15 SP5
 - Prometheus installation included
 - Certificate installation and https setup for nginx

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -41,6 +42,16 @@
...
      become: true
 
+- name: Provision prometheus
+  become: true
+  vars:
+   provision_prometheus: "true"
+  hosts: prometheus-hosts
+  roles:
+   - role: prometheus
+     when: provision_prometheus == 'true'
```
Comment (arbulu89) [inline playbook.yml:52]: we should document this `provision_prometheus` variable in the README file, together with the other `provision_` variables.
At the end, as prometheus is coming from non supported repos, we need to tell the user that they can even use their own prometheus installation

File: roles/proxy/tasks/main.yml [hunk 2]
```diff
@@ -8,6 +8,17 @@
...
 
+- name: Copy SSL certificate and key
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    owner: root
+    group: root
+    mode: "0644"
+  loop:
+    - { src: "{{ nginx_ssl_cert }}", dest: "/etc/ssl/certs/trento.crt" }
```
Comment (arbulu89) [inline roles/proxy/tasks/main.yml:19]: Wouldn't be better:
```
loop:
    - src: "{{ nginx_ssl_cert }}"
      dest: "/etc/ssl/certs/trento.crt"
    - src: "{{ nginx_ssl_key }}"
      dest: "/etc/ssl/private/trento.key"
```

Anyway, simply cosmethics

**Review by arbulu89 — COMMENTED** [review]
I have not been really involved, but some comments from my side anyway

File: roles/containers/defaults/main.yml [hunk 3]
```diff
@@ -30,3 +30,4 @@ web_admin_username: admin
 enable_alerting: "false"
 enable_api_key: "true"
 amqp_protocol: amqp
+prometheus_url: "http://host.docker.internal:9090"
```
[the remark this answers is not in the document]
Comment (rtorrero) [reply roles/containers/defaults/main.yml:33]: from the containers perspective, `host.docker.internal` references stuff that is running on the host machine. `localhost:9090` would point to the container itself (where only either web or wanda are running)

File: README.md [hunk 4]
```diff
@@ -277,12 +281,14 @@ These variables are the defaults of our roles, if you want to override the prope
...
 | nginx_conf_filename | Nginx vhost filename. "conf" suffix is added to the given name | trento |
-| nginx_vhost_listen_port | Configure the listen port for nginx trento server block | 80 |
+| nginx_vhost_http_listen_port | Configure the http listen port for trento (redirects to https by default) | 80 |
+| nginx_vhost_https_listen_port | Configure the https listen port for trento | 443 |
 | enable_api_key | Enable/Disable API key usage. Mostly for testing purposes | true |
 | web_upstream_name | Web nginx upstream name | web |
 | wanda_upstream_name | Wanda nginx upstream name | wanda |
 | grafana_upstream_name | Grafana nginx upstream name | grafana |
 | amqp_protocol | Change the amqp protocol type | amqp |
+| prometheus_url | Base url of prometheus database | http://host.docker.internal:9090 |
```
[the remark this answers is not in the document]
Comment (rtorrero) [reply README.md:291]: you mean on the description or the variable name?

File: roles/prometheus/tasks/main.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+- name: Create Prometheus system user
+  ansible.builtin.user:
+    name: prometheus
+    system: true
+    group: prometheus
+    shell: /sbin/nologin
+    comment: "User for Prometheus service"
+
+- name: Install package using expect
```
[the remark this answers is not in the document]
Comment (rtorrero) [reply roles/prometheus/tasks/main.yml:16]: I did, couldn't make it work :/

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (arbulu89) [reply README.md:291]: yes

**Comment by CDimonaco** [comment]
Green light when @arbulu89 review is addressed.

Good job

File: README.md [hunk 6]
```diff
@@ -150,7 +150,9 @@ all:
     "rabbitmq_username": "trentoansible",
     "prometheus_url": "http://localhost",
     "web_admin_password": "adminpassword",
-    "trento_server_name": "your-servername.com"
+    "trento_server_name": "your-servername.com",
+    "nginx_ssl_cert": "ssl/trento.crt",
```
Comment (CDimonaco) [inline README.md:153]: Please update with an example of certificate/key

**Review by CDimonaco — APPROVED** [review]
LGTM just the comment on the docs

**Review by arbulu89 — APPROVED** [review]

**Review by EMaksy — APPROVED** [review]
Nice work :+1:

**Pull request merged**